### ISSUE TITLE
SecureDrop Server can't send message to SecureDrop client

### ISSUE THREAD

**Issue by Harry-LeBlanc** [Matthew-Beliveau/SecureDrop#18]
The server can receive a message from a client, but it can't send a message back to the client. The issue can probably be resolved by specifying the ports in the send(msg) function and perhaps by using socket.detach() instead of socket.close().
